src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
```

Comment:
```suggestion
  constructor(health = 100, name) {
```
You can set a default value just once here